app/views/layouts/application.html.erb

```diff
@@ -11,6 +11,13 @@
   </head>
 
   <body>
+    <% if user_signed_in? %>
+      <%= link_to "Log out", destroy_user_session_path, data: { turbo_method: :delete } %>
+    <% else %>
+      <%= link_to "Log in", new_user_session_path %>
+    <% end %>
```

Comment:
14-18 provide a very simple log in/out link on the page. No styling at the moment, so just at the top of the page.